src/apify/_actor.py

```diff
@@ -55,9 +55,6 @@
 class _ActorType:
     """The class of `Actor`. Only make a new instance if you're absolutely sure you need to."""
 
-    _apify_client: ApifyClientAsync
-    _configuration: Configuration
-    _is_exiting = False
     _is_rebooting = False
```

Comment:
Why `is_rebooting` remain the class attribute and `is_exiting` not?

Reply:
Because when you call `reboot` on platform, it is an event that affects any possible `_ActorType` instances. `_is_exiting` can be local to the current instance, especially if we let users disable calling `sys.exit` from there.

Reply:
Thanks